docs/content/en/docs/clustermgmt/networking/networking-and-security.md

```diff
@@ -25,6 +25,26 @@ This section provides information about:
 * Using Cilium to securing workload connectivity using Kubernetes Network Policy.
 
 
+## Cilium Features
+The following table lists Cilium features and notes which of those features are built into EKS Anywhere, upstream Open Source, and Cilium Enterprise.
+
+<details><summary>Expand to see Cilium Features</summary>
+
+| Headline/Feature   | &nbsp;&nbsp;AWS EKS Anywhere | &nbsp;&nbsp;Cilium OSS |  &nbsp;&nbsp;Isovalent Cilium Enterprise |
```

Comment:
Bit of a nit but suggest using "EKS Anywhere" or "EKS Anywhere Default Cilium" as the col name

Reply:
Done.